bin/improver-ensemble-calibration

```diff
@@ -0,0 +1,128 @@
+                             'input phenomenon. This has to be supported by '
+                             'the minimisation functions in '
+                             'ContinuousRankedProbabilityScoreMinimisers.')
+
+    parser.add_argument('input_filepath', metavar='INPUT_FILE',
+                        help='A path to an input NetCDF file containing the '
+                             'current forecast to be processed.')
+    parser.add_argument('historic_filepath', metavar='HISTORIC_DATA_FILE',
+                        help='A path to an input NetCDF file containing the '
+                             'historic forecast used for calibration.')
+    parser.add_argument('truth_filepath', metavar='TRUTH_DATA_FILE',
+                        help='A path to an input NetCDF file containing the '
+                             'historic truth analyses used for calibration.')
+    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
+                        help='The output path for the processed NetCDF')
+
+    parser.add_argument('--predictor_of_mean', metavar='CALIBRATE_MEAN_FLAG',
+                        choices=['mean', 'members'], default='mean',
+                        help='String to specify the input to calculate the '
+                             'calibrated mean. Currently the ensemble mean '
+                             '("mean") and the ensemble members ("members") '
+                             'are supported as the predictors. Default: '
+                             '"mean".')
+    parser.add_argument('--save_mean_variance', metavar='MEAN_VARIANCE_FILE',
+                        default=False,
+                        help='Option to save output mean and variance from '
+                             'EnsembleCalibration plugin. If used, a path '
+                             'to save the output to must be provided.')
+    parser.add_argument('--num_members', metavar='NUMBER_OF_MEMBERS',
+                        default=None,
+                        help='Optional argument to specify the number of '
+                             'ensemble members to produce. Default will be '
+                             'the number in the raw input file.')
+    parser.add_argument('--random_ordering', default=False,
+                        action='store_true',
+                        help='Option to reorder the post-processed forecasts '
+                             'randomly. If not set, the ordering of the raw '
+                             'ensemble is used.')
+    args = parser.parse_args()
+    current_forecast = iris.load(args.input_filepath)
+    historic_forecast = iris.load(args.historic_filepath)
+    truth = iris.load(args.truth_filepath)
+    if not args.num_members:
+        args.num_members = len(current_forecast[0].coord('realization').points)
+
+    forecast_predictor_and_variance = EnsembleCalibration(
+        args.calibration_method, args.distribution, args.units,
+        args.predictor_of_mean).process(
```

Comment:
As `predictor_of_mean_flag` is a keyword argument, I'd change this to `predictor_of_mean_flag = args.predictor_of_mean`.